src/api-service/__app__/onefuzzlib/repro.py

```diff
@@ -85,6 +86,23 @@ def init(self) -> None:
 
                 self.state = VmState.extensions_launch
         else:
+            nsg = NSG(
+                name=vm.region,
+                region=vm.region,
+            )
+            result = nsg.create()
+            if isinstance(result, Error):
+                self.set_failed(result)
+                return
+
+            nsg_config = NetworkSecurityGroupConfig()
+            nsg_config.allowed_ips = ["*"]
```

Comment:
Do not forget to change this to set allowed IPs to values from Instance Config

Reply:
Updated.